EnrichmentFormSample/EnrichmentFormSample/Scripts/blades/edit-address.js

```diff
@@ -0,0 +1,245 @@
+
+            $scope.data = data;
+
+            var componentForm = {
+                street_number: 'short_name',
+                route: 'long_name',
+                locality: 'long_name',
+                administrative_area_level_1: 'short_name',
+                country: 'long_name',
+                postal_code: 'short_name'
+            };
+
+            if (!window.google || !window.google.maps) {
+                settings.getValues({ id: 'EnrichmentFormSample.General.GoogleMapApiKey' }).$promise.then(
+                    function (data) {
+                        var key = data[0];
+                        if (!key) {
+                            alert('You should set google api map key in General settings !');
+                            $scope.bladeClose();
+                            return;
+                        }
+                        $.getScript("https://maps.googleapis.com/maps/api/js?key=" + key + "&libraries=places&callback=MapApiLoaded",
+                            function () { });
+                        window.MapApiLoaded = function () {
+                            init();
+                        };
+                    });
+
+            }
+
+            function init() {
+                if (isInit) {
+                    return;
+                }
+                blade.title = blade.currentEntity.name;
+                blade.subtitle = 'edit-address.edit-address';
+
+                for (var i = 0; i < blade.currentEntity.properties.length; i++) {
+                    var property = blade.currentEntity.properties[i];
+                    var value = property.values.length ? property.values[0].value : '';
+                    switch (property.name) {
+                        case 'StreetAddress':
+                            data.streetAddress = value;
+                            break;
+                        case 'State':
+                            data.state = value;
+                            break;
+                        case 'Country':
+                            data.country = value;
+                            break;
+                        case 'City':
+                            data.city = value;
+                            break;
+                        case 'Position':
+                            if (value) {
+                                var pos = value.split(',');
+                                data.position = { lat: Number(pos[0]), lng: Number(pos[1]) };
+                            } else {
+                                data.position = null;
+                            }
+                            break;
+                        case 'Zip':
+                            data.zip = value;
+                            break;
+                    }
+                }
+
+                $scope.blade.isLoading = false;
+                initAutocomplete();
+                var mapCenter = data.position ? data.position : { lat: -33.8688, lng: 151.2195 };

```

Comment:
data.position ?? : { lat: -33.8688, lng: 151.2195 };

Reply:
JS don't have ?? it's only in C#.
But i'll  rewrite 
`data.position || { lat: -33.8688, lng: 151.2195 }`